tests/kuttl/tests/run_simple_playbook_debug/01-assert.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Check for:
+#
+# - 1 OpenStackAnsibleEE CR
+# - 1 Ansibleee-play pod
+# - 1 Ansibleee-play job
+# - Correct output from ansible play
+#
+apiVersion: ansibleee.openstack.org/v1alpha1
+kind: OpenStackAnsibleEE
+metadata:
+  name: ansibleee-play-debug
+  namespace: openstack
+spec:
+  image: quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
+  name: openstackansibleee-debug
+  play: |
+    - name: Print hello world
+      hosts: localhost
+      tasks:
+      - name: Using debug statement
+        ansible.builtin.debug:
+          msg: "Hello, world this is ansibleee-play-debug.yaml"
+status:
+  JobStatus: Active
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: openstack
+status:
+  phase: Active
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: openstackansibleee-debug
+    job-name: ansibleee-play-debug
+  name: ansibleee-play-debug
+  namespace: openstack
+status:
+  active: 1
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: |
+      pod=$(oc get pods -n openstack -l app=openstackansibleee-debug -o name)
```

Comment:
Same as above, in my environment pod is deployed with `app=openstackansibleee`

Reply:
How should I get it to deploy with some other app value?

Reply:
I'm not sure if that is configurable, I think it's set by the controller: https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/cf99bd1ef2d9aba2606266ca5684a31427123201/controllers/openstack_ansibleee_controller.go#L379

Reply:
Ok. That does make sense. I've adjusted the test not to look for it.